### PR TITLE
Bootstrap Layout

### PR DESCRIPTION
**Pull request by RichardSouta**

### CODE REVIEW

File: app/Presenters/templates/@layout.latte [hunk 1]
```diff
@@ -26,10 +26,48 @@
...
+                        </ul>
+                    </li>
+                </ul>
+
+            </div>
+        </div>
+    </nav>
+
+
+        <div class="jumbotron">
```
Comment (richard-ejem) [inline app/Presenters/templates/@layout.latte]: Please fix indentation to 1-level indent like nav tag above

File: app/Presenters/templates/@layout.latte [hunk 2]
```diff
@@ -26,10 +26,48 @@
...
+            <h1> Welcome to Instante</h1>
+            <p>Instante skeleton is ready, you can start creating your application now.
+        </div>
+
     {include #content}
 </div>
 {control jsLoader}
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (richard-ejem) [inline app/Presenters/templates/@layout.latte]: please do not use cdn bootstrap:
- inject JsModuleContainer to BasePresenter and `$this->jsModuleContainer->useModule('bootstrap3');` to BasePresenter::beforeRender
- add reference to bootstrap 3 to require-config.js: `bootstrap3: '../../bower_components/bootstrap/dist/js/bootstrap.min',`
- compile frontend using "gulp" command inside frontend folder

Comment (RichardSouta) [reply app/Presenters/templates/@layout.latte]: WIP: Please don't merge yet.

**Comment by RichardSouta** [comment]
WIP: Please don't merge yet.

**Comment by richard-ejem** [comment]
Re-factored&merged locally

**Pull request closed without merging**